### ISSUE TITLE
JSON incompatibility: Infinity and NaN

### ISSUE THREAD

**Issue by csimi** [philbooth/bfj#64]
It looks like these are not converted to null as in the native JSON.stringify.

https://tools.ietf.org/html/rfc4627
> 2.4.  Numbers:
> "Numeric values that cannot be represented as sequences of digits (such as Infinity and NaN) are not > permitted."

You can easily reproduce the error with this:

```javascript
bfj.stringify([Infinity, NaN]).then(JSON.parse).catch(console.trace);
```

**Comment by philbooth** [comment]
Thanks @csimi, I'll look into this at the weekend!

**Comment by philbooth** [comment]
This should be fixed in version `6.1.1`, released just now.

Also, note that I've moved this repo over to GitLab, so any problems with the above then you can continue the discussion in the equivalent issue over there:

https://gitlab.com/philbooth/bfj/issues/64

Thanks!
